components/dashboard/src/settings/Preferences.tsx

```diff
@@ -30,6 +30,14 @@ export default function Preferences() {
         const settings = additionalData.ideSettings || {};
         settings.defaultIde = value;
         additionalData.ideSettings = settings;
+        getGitpodService().server.trackEvent({
+            event: "ide_configuration_changed",
+            properties: {
+                useDesktopIde,
+                defaultIde: value,
+                defaultDesktopIde
+            },
+        });
```

Comment:
I'm seeing duplicate tracking events, one for `dashboard_clicked` and one for `ide_configuration_change`, do we want to remove the `dashboard_clicked` event and consolidate? Or is there value in having both? 

CC: @jakobhero / @gitpod-io/engineering-meta / @jldec

Reply:
Yeah, I agree, and I don't think this comment needs to block the merging of the PR, but would be curious to hear Jakob and/or team meta's opinion, as they know more about the purpose of the `dashboard_clicked` event.

Reply:
`dashboard_clicked` consistently tracks all user clicks happening on the dashboard - in combination with `page` and `path_changed` calls, it allows us to analyse the users' journeys throughout the dashboard. generally, we complement these events with what's happening on the server to enable the user (e.g. user clicks on `Create Team` button -> `dashboard_clicked` is fired, team is instantiated on the server -> `team_created` is fired). Here, it's fair to argue that the user's click makes modifications to his profile that should be tracked in its own call, but i do not recommend replacing the existing `dashboard_clicked` call. When we're analysing user flows, we're expecting that every meaningful click is tracked with a `dashboard_clicked` event, so this would introduce an exception that we would always have to consider when doing analysis.
Quick note on how to handle that `dashboard_clicked` is not fired for one element / component. Removing the track call in `Analytics.tsx` would remove it for all elements in the dashboard. Instead of doing this, pass `data-analytics = '{"dnt":true}'` to the HTML element or any of its ancestors to communicate to the `trackButtonOrAnchor` method that the element and/or any of children should not be tracked (see [here](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/src/Analytics.tsx#L75)). But again, I do not recommend to do this in this instance.

Reply:
Thanks for the explanation @jakobhero ! 🙏